products/sle/main.pm

```diff
@@ -500,7 +504,7 @@ sub load_inst_tests() {
     }
     loadtest "installation/addon_products_sle.pm";
     if (noupdatestep_is_applicable) {
-        if (check_var('ARCH', 'x86_64') && version_at_least('12-SP2') && is_server) {
+        if (check_var('ARCH', 'x86_64') && version_at_least('12-SP2') && is_server && (!is_sles4sap || is_sles4sap_standard)) {
```

Comment:
SLES4SAP should not allow users to install KVM/XEN in case of SLES4SAP installation (and should allow this in case of SLES installation), see https://bugzilla.suse.com/show_bug.cgi?id=980567